reference_data/management/commands/update_gene_shet.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from reference_data.management.commands.utils.update_utils import GeneCommand, ReferenceDataHandler
+from reference_data.models import GeneShet
+
+logger = logging.getLogger(__name__)
+
+
+class ShetReferenceDataHandler(ReferenceDataHandler):
+
+    model_cls = GeneShet
+    # The .tsv file is generated from the Google Doc at https://docs.google.com/spreadsheets/d/1enxGBWCAFBHdrRlqCj_ueleiDo9K9GWn/edit#gid=1146995171
+    # by downloading with a tsv format.
+    url = 'https://storage.googleapis.com/seqr-reference-data/gene_constraint/shet_Zeng(2023).xlsx%20-%20All%20scores-for%20gene%20page.tsv'
+
+    @staticmethod
+    def parse_record(record):
+        yield {
+            'gene_id': record['ensg'],
+            'shet': float(record['post_mean (Shet)']),
```

Comment:
It is the column name of the spreadsheet. I don't know what name is more suitable. Maybe we should ask Lynn.

Reply:
the column name is `post_mean (Shet)`. The name of this method is Shet. Therefore, this column is representing the post_mean score for the Shet method. Since the name of the table is Shet, we do not need to capture Shet in the name of the score in the table. Therefore, the name of the score in the table should be post_mean